envi/archs/rx/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+rxv2 module
+"""
+import envi
+
+from . import regs
+from . import const
+from . import disasm
+
+class RxModule(envi.ArchitectureModule):
+
+    def __init__(self):
+        envi.ArchitectureModule.__init__(self, "rx", maxinst=4)
```

Comment:
So...nobody actually every uses the maxinst data. Do you remember what it's for?